src/components/WwwFrame/DisclaimersContentful.vue

```diff
@@ -86,11 +91,13 @@ export default {
 
 				// gather all inactive promo banners by their start and end dates
 				const inactivePromoBanners = uiGlobalPromoSetting?.fields?.content?.filter(promoContent => {
+					// exclude items that are not global-promo-banners
+					const isGlobalPromo = promoContent?.sys?.contentType?.sys?.id === 'globalPromoBanner';
+					if (!isGlobalPromo) return false;
```

Comment:
We had someone trying to add an image in contentful which was blowing things up.